mlir/lib/Dialect/Rock/Tuning/GridwiseGemmParams.cpp

```diff
@@ -396,137 +396,137 @@ LogicalResult PopulateParamsAccel::obtainTuningParameters(
+  {64, 128, 2, 32, 64, 8, 1, true, true},
+  {64, 128, 2, 32, 64, 4, 1, true, true},
+  {64, 64, 8, 32, 32, 4, 1, true, true},
+  {64, 64, 4, 32, 32, 8, 1, true, true},
+  {64, 64, 4, 32, 32, 4, 1, true, true},
+  {64, 64, 4, 16, 64, 8, 1, true, true},
+  {64, 64, 4, 16, 64, 4, 1, true, true},
+  {64, 64, 2, 32, 32, 8, 1, true, true},
+  {32, 128, 4, 32, 32, 4, 1, true, true},
+  {32, 64, 4, 32, 64, 4, 1, true, true},
+  {32, 64, 4, 8, 64, 4, 1, true, true},
+  {32, 64, 8, 32, 32, 4, 1, true, true},
+  {32, 64, 8, 8, 64, 4, 1, true, true},
+  {32, 64, 2, 32, 32, 8, 1, true, true},
+  {32, 64, 2, 8, 64, 4, 1, true, true},
+  {32, 32, 8, 16, 16, 8, 1, true, true},
+  {32, 32, 8, 16, 16, 4, 1, true, true},
+  {32, 32, 4, 16, 16, 8, 1, true, true},
+  {16, 64, 8, 16, 16, 8, 1, true, true},
+  {16, 64, 8, 16, 16, 4, 1, true, true},
+  {16, 32, 8, 16, 16, 8, 1, true, true},
+  {16, 32, 8, 16, 16, 4, 1, true, true},
+  {16, 16, 8, 16, 16, 8, 1, true, true},
+  {16, 16, 4, 16, 16, 4, 1, true, true},
+  {8, 64, 8, 4, 64, 8, 1, true, true},
+  {4, 64, 16, 4, 64, 1, 1, true, true},
+  {4, 64, 2, 4, 64, 8, 1, true, true}
 };
 
 const InitParamsAccel
 PopulateParamsXDL::initParametersFp16[PopulateParamsXDL::nInitParametersFp16] = {
-  // M/block N/block K/block M/wave N/wave kPack forceUnroll bCopyMore
-  {128, 256, 8, 64, 128, 4, true, true},
-  {128, 256, 4, 64, 128, 8, true, true},
-  {128, 256, 2, 64, 128, 8, true, true},
-  {128, 256, 2, 64, 128, 4, true, true},
-  {128, 128, 4, 64, 64, 8, true, true},
-  {128, 128, 8, 64, 128, 1, true, true},
-  {128, 128, 8, 64, 64, 1, true, true},
-  {128, 128, 8, 64, 64, 4, true, true},
-  {128, 128, 4, 64, 64, 8, true, true},
-  {128, 128, 4, 64, 64, 4, true, true},
-  {128, 128, 2, 64, 128, 8, true, true},
-  {128, 128, 2, 64, 64, 8, true, true},
-  {64, 256, 2, 64, 64, 8, true, true},
-  {64, 128, 8, 32, 64, 4, true, true},
-  {64, 128, 4, 32, 64, 4, true, true},
-  {64, 128, 4, 32, 64, 8, true, true},
-  {64, 128, 2, 64, 64, 8, true, true},
-  {64, 128, 2, 32, 64, 8, true, true},
-  {64, 64, 8, 32, 32, 8, true, true},
-  {64, 64, 8, 32, 32, 4, true, true},
-  {64, 64, 8, 16, 64, 8, true, true},
-  {64, 64, 8, 16, 64, 4, true, true},
-  {64, 64, 4, 32, 32, 8, true, true},
-  {64, 64, 4, 16, 64, 8, true, true},
-  {64, 64, 4, 16, 64, 4, true, true},
-  {32, 128, 4, 32, 32, 8, true, true},
-  {32, 64, 8, 16, 64, 8, true, true},
-  {32, 64, 4, 32, 64, 4, true, true},
-  {32, 64, 4, 32, 32, 8, true, true},
-  {32, 64, 4, 16, 64, 8, true, true},
-  {32, 64, 2, 32, 64, 8, true, true},
-  {32, 32, 8, 16, 16, 8, true, true},
-  {32, 32, 4, 16, 16, 4, true, true},
-  {16, 64, 8, 16, 16, 8, true, true},
-  {16, 64, 4, 8, 64, 4, true, true},
-  {16, 64 ,4, 16, 16, 8, true, true},
-  {16, 32, 4, 16, 16, 8, true, true},
-  {16, 32, 4, 16, 16, 4, true, true},
-  {16, 16, 4, 16, 16, 8, true, true},
-  {8, 64, 8, 8, 64, 8, true, true}
+  // M/block N/block K/block M/wave N/wave kPack splitKFactor forceUnroll bCopyMore
```

Comment:
I'd like these to match the order they appear in in the perf config string (I think they do, just making sure)

Reply:
So, `forceUnroll, bCopyMore` are optional for `NonAccel` type. Therefore, I decided to move `splitKFactor` to the 7th place because it appears in both `Accel` and `NonAccel` variants.